vgen/src/main/java/io/gapi/vgen/go/GoDiscoveryContext.java

```diff
@@ -0,0 +1,137 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.go;
+
+import com.google.api.Service;
+import com.google.common.collect.ImmutableMap;
+import com.google.protobuf.Field;
+import com.google.protobuf.Method;
+import com.google.protobuf.Type;
+
+import io.gapi.vgen.ApiaryConfig;
+import io.gapi.vgen.DiscoveryContext;
+
+public class GoDiscoveryContext extends DiscoveryContext {
+  /**
+   * Constructs the Go language provider.
```

Comment:
Is this comment correct? Looks like the GoDiscoveryLanguageProvider instantiates a GoLanguageProvider and a GoDiscoveryContext.